packages/rich-text/src/internal/types/editor.ts

```diff
@@ -2,14 +2,15 @@
  * https://plate.udecode.io/docs/typescript
  */
 import { MARKS } from '@contentful/rich-text-types';
-import * as p from '@udecode/plate-core';
+import * as p from '@udecode/plate-common';
 import * as s from 'slate';
 import * as sr from 'slate-react';
-import {
-  SelectionMoveOptions as SlateSelectionMoveOptions,
-  SelectionCollapseOptions as SlateSelectionCollapseOptions,
-} from 'slate/dist/transforms/selection';
-import { TextInsertTextOptions as SlateTextInsertTextOptions } from 'slate/dist/transforms/text';
+// TODO fix me
```

Comment:
What's with this one?

Reply:
The type seems to be no longer exported in the later version, still not sure how to handle this

Reply:
import type { SelectionMoveOptions as SlateSelectionMoveOptions } from 'slate/dist/interfaces/transforms/selection'